cv32e40p/bsp/Makefile

```diff
@@ -6,7 +6,7 @@ RISCV_AR = $(RISCV_EXE_PREFIX)ar
 SRC = crt0.S handlers.S syscalls.c vectors.S
 OBJ = crt0.o handlers.o syscalls.o vectors.o
```

Comment:
> Looks good as long as you stay in master branch ...

Does this mean the 40p does not plan on staying in sync with "master"?

Reply:
It appeared that master and cv32e40p/dev were not in sync when we started working on it.
Discussed about that with Mike yesterday to fix it.

Reply:
I need to "merge down" to sync up cv32e40p/dev.